WordPress/Classes/ViewRelated/Comments/Views/Detail/CommentContentTableViewCell.swift

```diff
@@ -31,39 +30,37 @@ class CommentContentTableViewCell: UITableViewCell, NibReusable {
         }
     }
 
-    /// When supplied with a non-empty string, the cell will show a badge label beside the name label.
-    /// Note that the badge will be hidden when the title is nil or empty.
     var badgeTitle: String? = nil {
         didSet {
-            let title: String = {
-                if let title = badgeTitle {
-                    return title.localizedUppercase
-                }
-                return String()
-            }()
-
-            badgeLabel.setText(title)
+            let title = badgeTitle?.localizedUppercase ?? ""
+            badgeLabel.text = title
             badgeLabel.isHidden = title.isEmpty
-            badgeLabel.updateConstraintsIfNeeded()
         }
     }
 
-    override var indentationWidth: CGFloat {
-        didSet {
-            updateContainerLeadingConstraint()
-        }
+    var depth: Int {
+        get { effectiveDepth }
```

Comment:
Renamed from `indentationWidth` because it was one of the native `UITableViewCell` properties, which was a bit confusing. The new `effectiveDepth` is there to reduce the amount of unnecessary work.